src/FakeItEasy/Creation/DummyValueResolver.cs

```diff
@@ -268,7 +270,7 @@ private class ResolveByInstantiatingClassUsingDummyValuesAsConstructorArgumentsS
                 // of constructors enumerable.
                 var consideredConstructors = new List<ResolvedConstructor>();
 
-                if (this.cachedConstructors.TryGetValue(typeOfDummy, out ConstructorInfo cachedConstructor))
+                if (this.cachedConstructors.TryGetValue(typeOfDummy, out ConstructorInfo? cachedConstructor))

```

Comment:
no need to change but I'd've guessed `out var`

Reply:
I went for minimal changes, I guess